packages/frontend-2/plugins/020-rum.ts

```diff
@@ -126,18 +126,27 @@ async function initRumServer(app: PluginNuxtApp) {
     app.hook('app:rendered', (context) => {
       context.ssrContext!.head.push({
         script: [
+          {
+            innerHTML: `
+              new PerformanceObserver((list) => {
```

Comment:
w/ `?cwv=1` this will also now emit all LCP candidate entries allowing for easier LCP score troubleshooting (the last/biggest entry reported is the final one)